plugin/server/datastore-sqlite/sqlite.go

```diff
@@ -905,12 +1101,13 @@ func (ds *sqlitePlugin) restart() error {
 		ds.db.Close()
 	}
 
+	db.Exec("PRAGMA foreign_keys = ON")
```

Comment:
For the record, this is the line we needed to get foreign keys working. In addition, GORM does not automatically install foreign key constraints on tables it creates with AutoMigrate - instead, we must manually specify the constraints in an anchor on the model's field like this: https://github.com/spiffe/spire/pull/337/files#diff-388f59d699e88e5a586dea9075f6129dR15